.changeset/loud-comics-search.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/plugin-tech-insights': minor
+'@backstage/plugin-tech-insights-backend-module-jsonfc': minor
+'@backstage/plugin-tech-insights-backend': patch
+'@backstage/plugin-tech-insights-common': patch
```

Comment:
I think it's best to break this up into multiple changesets that target the packages more specifically. As it is atm each changelog of these packages will get the full description below